lib/tool_shed/webapp/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "@galaxyproject/tool-shed",
+  "version": "1.0.0",
```

Comment:
Should we follow the Galaxy version here ?

Reply:
https://github.com/galaxyproject/galaxy/blob/dev/package.json#L3 - ummm this is what Galaxy uses for the comparable file.

Reply:
I don't remember why I did that for the client installation package -- might not have been intentional, just what the package scaffold spit out :)

The galaxy client itself (@galaxyproject/galaxy-client) should definitely be versioned along with the platform, and I see now that didn't make the 23.1 jump (I'm going to do that now), but I think how "@galaxyproject/tool-shed-frontend" versioned separate from the YYNN scheme is likely totally fine?  We won't have toolshed releases every galaxy release, even, right?

Reply:
It's also worth noting that the version in these 'installer' package.json files is not significant -- these are not published to npm and are only a local convenience measure that's always linked directly to the deployed code inherently.

Reply:
I guess I was thinking there's no harm in putting the publishing together with the release script, but no strong preference.